examples/beacon-api-sse/src/main.rs

```diff
@@ -19,10 +19,10 @@ use clap::Parser;
 use futures_util::stream::StreamExt;
 use mev_share_sse::{client::EventStream, EventClient};
 use reth::{
-    cli::{
+    cli::Cli,
+    node_core::cli::{
```

Comment:
ideally we have this not breaking,
I think this request caused the issues in the previous PR... 
but whatever, at this point I really don't care anymore

though the module should not be called node_core but core

Reply:
I can make the rename, and should be able to make it non-breaking, to make these import changes unnecessary